scanner/src/main/kotlin/Scanner.kt

```diff
@@ -172,18 +172,21 @@ abstract class Scanner(val scannerName: String, protected val config: ScannerCon
      * the [project]s definition file.
      */
     private fun filterProjectScanResults(project: Project, resultContainer: ScanResultContainer): ScanResultContainer {
+        var filteredResults = resultContainer.results
```

Comment:
Commit message: "As" in the beginning is unnecessary. Please reference the commit of the bug you mention.

Reply:
Done.